ambari-server/src/main/java/org/apache/ambari/server/stack/StackManager.java

```diff
@@ -569,8 +569,8 @@ private void validateExtensionDirectory(File extensionRoot) throws AmbariExcepti
           continue;
         }
         for (File serviceFolder : commonService.listFiles(StackDirectory.FILENAME_FILTER)) {
-          String serviceName = serviceFolder.getParentFile().getName();
-          String serviceVersion = serviceFolder.getName();
+          serviceFolder.getParentFile().getName();
+          serviceFolder.getName();
```

Comment:
I see the variables were unused.  Is the goal of these leftover calls to verify something?

Reply:
I think my IDE did this - I don't think these calls are for anything, so I'll remove them.